src/molecule/__init__.py

```diff
@@ -25,8 +25,8 @@
 __metaclass__ = type
 
 try:
-    import pkg_resources
+    import importlib_metadata
```

Comment:
This shouldn't be a part of try-except because it's clearly listed as a dependency.

Reply:
I do not fully remember all the reasons why we had this approach but I think that it was something related to running unittest without installing the package. I will try to remove and see what happens.